coordinator/internal/types/get_task.go

```diff
@@ -4,13 +4,16 @@ package types
 type GetTaskParameter struct {
 	ProverHeight uint64 `form:"prover_height" json:"prover_height"`
 	TaskTypes    []int  `form:"task_types" json:"task_types"`
+	TaskID       string `form:"task_id,omitempty" json:"task_id,omitempty"`
+	Universal    bool   `form:"universal,omitempty" json:"universal,omitempty"`
 }
 
 // GetTaskSchema the schema data return to prover for get prover task
 type GetTaskSchema struct {
 	UUID         string `json:"uuid"`
 	TaskID       string `json:"task_id"`
 	TaskType     int    `json:"task_type"`
+	UseSnark     bool   `json:"use_snark,omitempty"`
```

Comment:
seems nowhere uses it

Reply:
It should be specified by bundle task to obtain snark proof. "TaskType" is considered to be deprecated later since universal prover do not distinguish task type (it simply select the corresponding app from the `vk` which is specified in taskdata)